show/system_health.py

```diff
@@ -159,3 +167,93 @@ def sysready_status_detail():
         clicommon.run_command(cmd, display_cmd=False)
     except Exception as e:
         click.echo("Exception: {}".format(str(e)))
+
+
+def is_smartswitch():
+    return hasattr(device_info, 'is_smartswitch') and device_info.is_smartswitch()
+
+
+def show_module_state(module_name):
```

Comment:
@rameshraghupathy can we rename as show_dpu_state() since the KEY is hardcoded to DPU_STATE?

Reply:
@prgeor Done